tsconfig.json

```diff
@@ -11,7 +11,7 @@
       "declarationDir": "./dist/types",
       "esModuleInterop": true,
       "allowSyntheticDefaultImports": true,
-      "lib": ["ES2015"]
+      "lib": ["ES2015", "ESNext"]
```

Comment:
What the purpose of ESNext? 🙂

Reply:
This are the rules to transpile typescript to javascript. By saying `ESNext` I inform the transpiler that I will be writing with the last features of javascript. This will make it possible for me to write for example [optionnal chaining](https://github.com/tc39/proposal-optional-chaining). These features are not released in an official Ecmascript version but are bound to be in the next one. 
```
if (response.data?.hits?.something)
```

Reply:
Thanks for the explanations! 😁